cmd/opdoc/opdoc.go

```diff
@@ -252,42 +265,45 @@ type OpRecord struct {
 	Groups            []string
 }
 
+type namedType struct {
+	Name         string
+	Abbreviation string
+	Bound        []uint64
```

Comment:
```suggestion
	Bound        [2]uint64
```

Reply:
Oh, is this only used locally? (must be, lowercase) So yeah, I guess that's fine.

Wait, won't it only be two 0's for the constant 0? Almost seems better to show it for that rare case, since it's so meaningful.